python/cuml/neighbors/kneighbors_classifier.pyx

```diff
@@ -305,3 +305,10 @@ class KNeighborsClassifier(NearestNeighbors, ClassifierMixin):
 
     def get_param_names(self):
         return super().get_param_names() + ["weights"]
+
+    def _more_tags(self):
+        return {
+            # fit and predict require conflicting memory layouts
+            'preferred_input_order': None
```

Comment:
I think this could be fixed but I'll need to look into it. Created https://github.com/rapidsai/cuml/issues/3153